tests/unit/test_registry.py

```
@@ -114,3 +117,180 @@ def test_import_shim_returns_registry_class() -> None:
+@pytest.mark.parametrize(
+    ("api_function", "call_kwargs"),
+    [
+        (completion, {"messages": [{"role": "user", "content": "hi"}]}),
+        (embedding, {"inputs": "hi"}),
+        (moderation, {"input": "hi"}),
+        (image_generation, {"prompt": "a cat"}),
+    ],
+    ids=["completion", "embedding", "moderation", "image_generation"],
+)
+def test_api_entry_points_resolve_registry_only_rows(
+    community_row: OpenAICompatibleProviderConfig,
```

Ruff (0.16.1)

[warning] Unused function argument: `community_row`

(ARG001)